byway/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,7 +29,26 @@
 
```

Comment:
maybe call this servlet something else like RecomendationsServlet to so its more specific

Reply:
done.